Team8923_PowerPlay/src/main/java/org/firstinspires/ftc/team8923_PowerPlay/ConceptTensorFlowObjectDetectionWebcam.java

```diff
@@ -0,0 +1,153 @@
+package org.firstinspires.ftc.team8923_PowerPlay;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+import org.firstinspires.ftc.robotcore.external.ClassFactory;
+import org.firstinspires.ftc.robotcore.external.hardware.camera.WebcamName;
+import org.firstinspires.ftc.robotcore.external.navigation.VuforiaLocalizer;
+import org.firstinspires.ftc.robotcore.external.tfod.Recognition;
+import org.firstinspires.ftc.robotcore.external.tfod.TFObjectDetector;
+
+import java.util.List;
+@Autonomous(name = "ConceptTensorFlowObjectDetectionWebcam ")
+public class ConceptTensorFlowObjectDetectionWebcam extends BaseOpMode {
+
+    /*
+     * Specify the source for the Tensor Flow Model.
+     * If the TensorFlowLite object model is included in the Robot Controller App as an "asset",
+     * the OpMode must to load it using loadModelFromAsset().  However, if a team generated model
+     * has been downloaded to the Robot Controller's SD FLASH memory, it must to be loaded using loadModelFromFile()
+     * Here we assume it's an Asset.    Also see method initTfod() below .
+     */
+    private static final String TFOD_MODEL_ASSET = "RedTensorFlowModel.tflite";
```

Comment:
Why do you have a red and blue one? What was the one you were using at competition? That one was actually working. I feel like this code is not the code you were using at competition.

Reply:
Will replace with april tags